to.holepunch.Keet.yml

```diff
@@ -0,0 +1,52 @@
+id: to.holepunch.Keet
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+command: /app/bin/usr/bin/keet
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=all
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+  - --talk-name=org.freedesktop.ScreenSaver
+  - --filesystem=host
+modules:
+  - name: assets
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/share/metainfo/
+      - mkdir -p /app/share/icons/hicolor/512x512/apps/
+      - mkdir -p /app/share/applications/
+      - cp -r assets/. /app/bin/ 
+      - cp to.holepunch.Keet.metainfo.xml /app/share/metainfo/
+      - cp to.holepunch.Keet.png /app/share/icons/hicolor/512x512/apps/
+      - cp to.holepunch.Keet.desktop /app/share/applications/
+    sources:
+      - type: dir
+        path: .
+  - name: keet-x64
+    only-arches:
+      - x86_64
+    buildsystem: simple
+    build-commands:
+      - cp keet /app/bin/usr/bin/
+    sources:
+      - type: archive
+        url: https://keet.io/downloads/latest/keet-x64-binary.tar.gz
```

Comment:
this is an unversionned URL. Which mean the build WILL break. It should be a specific version.